functional/OpenJcePlusTests/build.xml

```diff
@@ -123,7 +123,8 @@
 	<target name="build" >
 		<if>
 			<or>
-				<contains string="${TEST_FLAG}" substring="OpenJcePlusTests"/>
+				<contains string="${TEST_FLAG}" substring="FIPS140_3_OpenJCEPlusFIPS"/>
+				<contains string="${TEST_FLAG}" substring="FIPS140_3_OpenJCEPlusFIPS.FIPS140-3"/>
```

Comment:
`<contains string="${TEST_FLAG}" substring="FIPS140_3_OpenJCEPlusFIPS"/>` is sufficent. We do not need the second line